apps/coscrad-frontend/src/components/commands/command-selection-buttons.tsx

```diff
@@ -0,0 +1,19 @@
+import { ICommandFormAndLabels } from '@coscrad/api-interfaces';
+import { CommandButton } from './command-button';
+
+interface Props {
```

Comment:
Should we just call the component interface Props?  Or stick to the <component name>Props format?

Reply:
Ah this was kind of lazy on my part. I'm not exporting it so I figured who cares about the name. I'll fix this.

Reply:
I've fixed this now.